src/Form/grid/Element/index.tsx

```diff
@@ -399,6 +399,7 @@ const Element = ({ node: el, form, flags }: any) => {
             setRef={(ref: any) => {
               if (firstField) focusRef.current = ref;
             }}
+            repeat={index}
```

Comment:
dropdowns aren't the only fields that use options. all option-based fields should be updated

Reply:
also repeatIndex might be a more accurate name here for the prop